apps/e2e/cypress/e2e/FAPs.cy.ts

```diff
@@ -24,6 +24,16 @@ const fapMembers = {
   reviewer2: initialDBData.users.user3,
 };
 
+function clickConfirmOk() {
```

Comment:
From what I can see, this puts the conflict-of-interest feature under a feature flag (which is what we discussed), but if that feature flag is enabled we still have a broken conflict-of-interest feature?

Reply:
That would be the case, yes. I thought we chose to turn it off for stfc then fix, was this not what we discussed?

Reply:
Yeah I think it was actually - from the meeting we had it doesn't sound like it'll ever actually be used, so I wouldn't be surprised if we just deleted it completely one day.